faq.md

```diff
@@ -31,10 +31,16 @@ _Open-source software steward_ is a term defined in [Article 3(14)][] of the CRA
 
 > ‘open-source software steward’ means a legal person, other than a manufacturer, that has the purpose or objective of systematically providing support on a sustained basis for the development of specific products with digital elements, qualifying as free and open-source software and intended for commercial activities, and that ensures the viability of those products;
 
+_Open-source software steward_ are subject to a "light-touch and tailor-made regulatory regime" ([Recital 19][]), defined in [Article 24][].
+
 #### Q: Can an solo maintainer be considered to be an _open-source software steward_?
 
+No. As defined in [Article 3(14)][], an _open-source software steward_ must be a _legal person_ (e.g. a company, an organization, etc.) in contrast with a _natural person_ (i.e. a human being). The obligations of _open-source software stewards_ described in [Article 24][] therefore do not apply to solo maintainers. It is worth noting however, that _natural persons_ are subject to the same obligations as _legal persons_ would be should they monetize their poject.
+
 #### Q: Can a loosely organized group of maintainers be considered to be an _open-source software steward_?
 
+In [Article 3(14)][], the CRA defines _open-source software steward_ as a _legal person_ which whould exempt loosely organized groups of maintainers from the obligations of _open-source software stewards_. However, in certain European jurisdictions, groups of people acting in concert have been considered as a _legal person_ in certain cases, even though their relationship wasn't formalized through by creating an organization of any kind. So it is possible that in some cases, a loosely organized group of maintainers might be considered to be an _open-source software steward_ and therefore subject to the obligations of _open-source software stewards_ described in [Article 24][]. It is worth noting that not complying to those obligations doesn't carry penalties as per [Article 64(10)(b)][].
```

Comment:
@maertsen I re-wrote this. LMK what you think.

Reply:
@tobie if the formulation excludes or includes loosely formed groups, is something we do not know. I would prefer to add the CAUTION marking here.

And add to the analysis that the steward has obligation to enforce certain things on the project. In my opinion, then this group will need to designate how they handle such enforcement, and what if people with actual write rights to the repository disagree? This is a subject for another question, I think. (it can also cover the case of foundation that have no right to enforce development decisions and handle legal or financial only - for example SPI)

Reply:
I played with the CAUTION label, but it made it stand out way more than desirable. Curious to hear what others think, here.

Agree with you that the other points you mention are fodder for other questions, but don't belong here.